### ISSUE TITLE
Tournament Support

### ISSUE THREAD

**Issue by regiszanandrea** [kevinohashi/php-riot-api#64]
Hello, 

Please, add support to Tournaments 👍  

Thanks.

**Comment by ArfaouiHamouda** [comment]
Hi , can't add something that Riot Basically doesn't provide in her API .

**Comment by regiszanandrea** [comment]
Yes, They provide: https://developer.riotgames.com/tournament-api.html

**Comment by ArfaouiHamouda** [comment]
Oh sorry my bad , didn't check it for a while .
